src/i18n/i18n.json

```diff
@@ -111,7 +111,14 @@
         "thu": "T",
         "fri": "F",
         "sat": "S"
-      }
+      },
+      "sun": "Sunday",
```

Comment:
These should be wrapped in a "long" object.
Just like month names.
Generally there are 3 types of names: long (full), short (3 letters) and min (1 letter).